benchmark/CMakeLists.txt

```diff
@@ -0,0 +1,50 @@
+# Link binary ${basename} against the threading library,
+# and all libraries that are specified as additional arguments.
+function (linkBenchmark basename)
+    target_link_libraries(${basename} absl::flat_hash_map ${ARGN} ${CMAKE_THREAD_LIBS_INIT})
+endfunction()
+
```

Comment:
Does your benchmark actually need the threading library, or did you just copy this from the tests (googletest needs the threading, Your benchmark not as far as I see).

Reply:
It doesn't, I just wasn't sure, if any of the benchmark files, that other people will write, will need it. Currently not file needs it, so I removed it in [b44abb2](https://github.com/ad-freiburg/qlever/pull/860/commits/b44abb2d8e813d5dd70b4828b24a618b792a2a16).